quickstarts/microsoft.sql/sqlmi-new-vnet-w-diagnostic-settings/metadata.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
check the pipeline results for issues that need to be fixed here

Reply:
Updated, tnx!